features/partners.feature

```diff
@@ -23,12 +21,12 @@ Functionaliteit: Huwelijken en geregistreerd partnerschappen van een persoon raa
       | naam                      | waarde   |
       | datum aangaan (06.10)     | 20010215 |
       | soort verbintenis (15.10) | H        |
-      | indicatie onjuist (84.10) |          |
+      En de partner heeft GEEN ontbindingHuwelijkPartnerschap gegevens
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                          |
       | type                | RaadpleegMetBurgerservicenummer |
       | burgerservicenummer | 999990184                       |
-      | fields              | partners.naam.voornamen         |
+      | fields              | partners                        |
```

Comment:
Met deze fields waarde wordt alle partners gegevens opgevraagd. De aangaagHuwelijkPartnerschap gegevens zitten ook in de response. Hoeft dat niet te worden gevalideerd?

Reply:
ik denk dat het doel van de features is om specifieke functionaliteit te beschrijven. Het leveren van gegevens (zoals beschreven in [/docs/BRP-LO GBA mapping.xlsx](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/master/docs/BRP-LO%20GBA%20mapping.xlsx) hebben we in geen enkele feature beschreven.